packages/prisma/seed-app-store.ts

```diff
@@ -337,7 +337,11 @@ export default async function main() {
       invite_link: process.env.ZAPIER_INVITE_LINK,
     });
   }
-
+  if (process.env.MAKE_INVITE_LINK) {
+    await createApp("make", "make", ["automation"], "make_automation", {
+      invite_link: process.env.MAKE_INVITE_LINK,
+    });
+  }
```

Comment:
We don't need to do this. Infact, we don't want to do this. Whether an app is enabled(and seeded) or not is controlled through the admin/apps interface.